kubernetes/provider.go

```diff
@@ -40,46 +40,57 @@ func Provider() *schema.Provider {
 	p := &schema.Provider{
 		Schema: map[string]*schema.Schema{
 			"host": {
```

Comment:
When i tried this out I was still able to set both `host` and `config_path`.

I think you need to set ConflictsWith for both attributes, i.e you need to add `host` to the ConflictsWith for `config_path`. You'll need to this for the other attributes you've added this too as well.